From e262f8fde26668988dd238e45c7a747f54778eec Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 21:26:05 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the direct GnuPG verification with %openpgpverify, which
uses Sequoia sqv, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh p11-kit-release-keyring.gpg from keyservers/WKD: same
certificate(s), with self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 p11-kit.spec | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/p11-kit.spec b/p11-kit.spec
index bbc8f07..ede86c9 100644
--- a/p11-kit.spec
+++ b/p11-kit.spec
@@ -33,6 +33,7 @@ BuildRequires:  pkgconfig(glib-2.0)
 BuildRequires:  pkgconfig(systemd)
 BuildRequires:  gnupg2
 BuildRequires:  /usr/bin/xsltproc
+BuildRequires:  openpgpverify
 
 %if %{with mingw}
 BuildRequires:  ninja-build
@@ -137,7 +138,7 @@ way that they're discoverable.  This library is cross-compiled for MinGW.
 
 
 %prep
-gpgv2 --keyring %{SOURCE2} %{SOURCE1} %{SOURCE0}
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 
 %autosetup -p1
 
-- 
2.56.0

